docs/database/database-spans.md

```diff
@@ -85,6 +87,58 @@ and SHOULD adhere to one of the following values, provided they are accessible:
 If a corresponding `{target}` value is not available for a specific operation, the instrumentation SHOULD omit the `{target}`.
 For example, for an operation describing SQL query on an anonymous table like `SELECT * FROM (SELECT * FROM table) t`, span name should be `SELECT`.
 
+## Status
+
+[Span Status Code][SpanStatus] MUST be left unset if the operation has ended without any errors.
+
+Instrumentation SHOULD consider the operation as failed if any of the following is true:
+
+- the `db.response.status_code` value indicates an error
+
+  > [!NOTE]
+  >
+  > The classification of status code as an error depends on the context.
+  > For example, a SQL STATE `no_data` status code indicates an error when the application
+  > expected the data to be available. However, it is not an error when the
+  > application is simply checking whether the data exists.
+  >
+  > Instrumentations that have additional context about a specific operation MAY use
+  > this context to set the span status more precisely.
+  > Instrumentations that don't have any additional context MUST follow the
+  > guidelines in this section.
+
+- an exception is thrown by the instrumented method call
+- the instrumented method returns error in another way
+
+When the operation ends with an error, instrumentation:
+
+- SHOULD set the span status code to `Error`
+- SHOULD set the `error.type` attribute
+- SHOULD set the span status description when it has additional information
+  about the error that aligns with [Span Status Description][SpanStatus] and
+  can't be inferred from `db.response.status_code` or `error.type` attributes.
```

Comment:
I didn't follow why this part? thanks

> and can't be inferred from `db.response.status_code` or `error.type` attributes

Reply:
I'm trying to avoid duplicating things between `db.response.status_code` / `error.type` and span status description.

We have a similar wording in HTTP 

https://github.com/open-telemetry/semantic-conventions/blob/43f9e308ada81c5ef6851cd665d95f294b157a0f/docs/http/http-spans.md?plain=1#L112

Let me see if I can explain it more clearly

Reply:
> We have a similar wording in HTTP

👍

I was thinking it might be useful for backends which display error status + description in a general way across all spans, but definitely understand the duplication concern

Reply:
I'm good resolving, makes sense to stay consistent with HTTP